src/command-line/index.js

```diff
@@ -39,6 +39,13 @@ require("./reset");
 require("./edit");
 require("./install");
 
+// TODO: Remove this when releasing The Lounge v3
+if (process.argv[1].endsWith("/lounge")) {
```

Comment:
Why `endswith`? Wouldn't that make it match "thelounge"? Also, is the `/` a typo?

Reply:
Slash is specifically so it doesn't match with `thelounge`. But does that slash have to match OS default one?

Reply:
@YaManicKill, the idea is that `process.argv[1]` always contains the full path, so I can't test for equality, hence the forward slash.

@xPaw, that's a good point! Would you mind running `console.log(process.argv[1])` for me and let me know what it tells you? I don't know if Node normalizes those or not.

I also don't want to do a `!.endsWith("thelounge")` because that would match when running `npm start`.
If `/` is an issue, I'll go with a `/(^the)lounge$/` regex (but I suck at regex so that might just be wrong, I'd need to play) or `.endsWith("lounge") && !.endsWith("thelounge")`. Gross, but it's a temporary check anyway so whatever.

Reply:
Use `${path.sep}lounge`.

Reply:
Ah, awesome! Done.